sdk/src/main/java/com/mapbox/maps/renderer/egl/EGLConfigChooser.kt

```diff
@@ -88,28 +86,48 @@ internal class EGLConfigChooser constructor(
 
   private fun getNumberOfConfigurations(
     egl: EGL10,
-    display: EGLDisplay,
-    configAttributes: IntArray
+    display: EGLDisplay
   ): IntArray {
     val numConfigs = IntArray(1)
-    if (!egl.eglChooseConfig(display, configAttributes, null, 0, numConfigs)) {
-      Logger.e(
-        TAG,
-        String.format(
-          MAPBOX_LOCALE,
-          "eglChooseConfig returned error %d",
-          egl.eglGetError()
+    val initialSampleCount = antialiasingSampleCount
+    while (true) {
```

Comment:
I'm a bit nervous to see the while(true) 😅

Reply:
pretty common practice when configuring EGL, e.g. see [here](https://github.com/brackeen/glfm/blob/0581711cc7a1173e5a63d2982d506ec97716b17f/src/glfm_platform_android.c#L633) 😄